src/components/ReportActionItem/ReportPreview.tsx

```diff
@@ -200,11 +201,13 @@ function ReportPreview({
         if (ReportUtils.hasHeldExpenses(iouReport?.reportID)) {
             setIsHoldMenuVisible(true);
         } else if (chatReport && iouReport) {
+            HapticFeedback.longPress();
             if (ReportUtils.isInvoiceReport(iouReport)) {
                 IOU.payInvoice(type, chatReport, iouReport, payAsBusiness);
             } else {
                 IOU.payMoneyRequest(type, chatReport, iouReport);
             }
+            return true;
```

Comment:
Regarding this [concern](https://github.com/Expensify/App/issues/48036#issuecomment-2329654361), I move the state whether to play the animation or not to AnimatedSettlementButton. To know whether pressing the pay button will pay the request or show a hold notification modal, I return true here.

Related to the hold notification modal, do we also want to play the pay animation when confirming paying from the hold notification modal?

This is how it look now.

https://github.com/user-attachments/assets/21cee708-b4a2-466e-abd3-a145f9ccc98a